src/App/Tracker/Controller/Hooks/Listeners/JoomlacmsPullsListener.php

```diff
@@ -738,7 +738,7 @@ protected function checkCategories($hookData, Github $github, Logger $logger, $p
 	 */
 	protected function checkFilesAndAssignCategory($files)
 	{
-		$addCategories = [];
+		$addCategories = array();
```

Comment:
So since it's "the same" - why did you changed it? :wink: (sorry for the late reply :tongue:)